pkg/drivers/mysql/mysql.go

```diff
@@ -34,10 +35,12 @@ var (
 				PRIMARY KEY (id)
 			);`,
 	}
-	nameIdx     = "create index kine_name_index on kine (name)"
-	nameIDIdx   = "create index kine_name_id_index on kine (name,id)"
-	revisionIdx = "create unique index kine_name_prev_revision_uindex on kine (name, prev_revision)"
-	createDB    = "create database if not exists "
+	nameIdx      = "create index kine_name_index on kine (name)"
+	nameIDIdx    = "create index kine_name_id_index on kine (name,id)"
+	deletedIDIdx = "create index kine_id_deleted_index on kine (id,deleted)"
+	prevRevIdx   = "create index kine_prev_revision_index on kine (prev_revision)"
+	revisionIdx  = "create unique index kine_name_prev_revision_uindex on kine (name, prev_revision)"
```

Comment:
nit: can we change the name here to reflect the index name i.e (prevRevNameIdx)

Reply:
I didn't actually change that one, it just got some whitespace added by gofmt so that the equals signs all line up. The mysql driver is the only one that has separate variables for the indexes - it doesn't support 'IF NOT EXISTS' like the other engines do so we have to handle and ignore the resulting errors instead of just blind executing the DDL.

Reply:
Fixed it anyway. This extra bit was actually unnecessary; we can just handle schema creation like we do in the other drivers, but with special handling for the duplicate key error that is raised if we try to create an index that already exists.